src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -294,3 +296,12 @@
                                                 {:event  :wallet/send-transaction
                                                  :error  error
                                                  :params request-params}))}]})))
+
+(rf/reg-event-fx
+ :wallet/send-flow-navigate-forward
+ (fn [_ [{:keys [current-screen start-flow?]}]]
+   {:fx [[:dispatch
+          [:navigation/wizard-forward
+           {:current-screen current-screen
+            :start-flow?    start-flow?
+            :flow-config    flow-config/steps}]]]}))
```

Comment:
Do we need this intermediary event? Why can't we directly call `:navigation/wizard-forward`

Reply:
It seems we are creating so many wrappers for dx.

:wallet/send-flow-navigate-forward -> :navigation/wizard-forward -> Actual navigation events -> Then core navigation

Reply:
Can we create a single file for storing flow-config?

Reply:
In common/wizard itself. Then access this config in wizard.cljs? (By passed key etc.) (Means which wizard flow we are using)

Reply:
> It seems we are creating so many wrappers for dx.
> 
> 
> 
> :wallet/send-flow-navigate-forward -> :navigation/wizard-forward -> Actual navigation events -> Then core navigation

This is true but there is 2 other flows (and more in the future) that will use this, each used in several points across wallet.

Reply:
> In common/wizard itself. Then access this config in wizard.cljs? (By passed key etc.) (Means which wizard flow we are using)

I would say no. The flow definition should live close to its usage.
I shouldn't have to look in status-im/common... to find out where the navigation of the wallet send flows or bridge flows are defined.
Perhaps I have misunderstood what you meant. 🤔

Reply:
Thank you for explanation. If this way make more sense please feel free to keep it.

But usually, when we have different wrapper they are doing different things. For example, `:chat/navigate-to-chat` is very special case, and we can't duplicate that event.

`:wallet/send-flow-navigate-forward` is not special, other flows will be exactly same (if I am not wrong), and will just call `:navigation/wizard-forward` with only change of `flow-config/steps`.

> Yeah we could do that but it's better the flow lives close to it's use imo.

Yes, we can still keep that. We can just import these flows in wizard.cljs, and instead of passing this flow-config, we just pass flow-id. And let `wizard-find-next-screen` handle all the magic. wdyt?

Reply:
Sure, let's try that 👍

Reply:
Done